fix(preview): handle libexif missing fields, don't accept 0-value orientation

anthonybilinski opened the pull request:
Last fix was a hack when the error was actually us using the default 0 value of
orientation since the returned ExifData was null.

- [x] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qtox/qtox/6168)
<!-- Reviewable:end -->

anthonybilinski committed:
fix(preview): handle libexif missing fields, don't accept 0-value orientation

Last fix was a hack when the error was actually us using the default 0 value of
orientation since the returned ExifData was null.

sudden6 reviewed (approved):
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/6168#-:-M9F69viAPKwXTZLjluG:b-jbsf8z)* status: :shipit: complete! 1 of 1 LGTMs obtained



<!-- Sent from Reviewable.io -->

anthonybilinski committed:
fix(core): read proxy type after personal settings are loaded

Profile is required to load personal settings, and Profile was creating the
bootstraplist using personal settings. Instead create the bootstraplist in
initCore, after personal settings have been loaded. This avoids using an
uninitialized proxy type on every profile load.
